style.css

```diff
@@ -1039,8 +1054,8 @@
   }
 
   .form-section p {
-    margin-left: -263px;
-    margin-right: -307px;
+    margin-left: 23%;
+    margin-right: 23%;
```

Comment:
- [ ]  Great job fixing the margins as requested by the previous reviewer, however, as mentioned by the previous reviewer, when I reach the 768px break point the contact form is still not correctly aligned like in the Figma template. The paragraph from this section also doesn't display correctly.

Your design:
![contactcentered](https://user-images.githubusercontent.com/111214189/229266581-50f7a589-fb84-4771-adbb-5fa2ca402a94.png)

Figma template: 
![templatece](https://user-images.githubusercontent.com/111214189/229266680-6cede9e9-7bbf-4a9c-a1ed-31d8576947f2.png)

Kindly fix this issue 👍 . Making your project responsive, will help your portfolio look more professional and show that you have great attention to detail.

(Hint: you can take a look at the negative margin values used in your .form-section h1 and the padding-inline used in the .form-section in the media queries. Make sure to check that the form is correctly aligned on mobile and desktop and that is responsive)

Reply:
I think i have it.
![image](https://user-images.githubusercontent.com/111823606/229287510-1fa61892-41da-463a-beba-b38e31121707.png)